packages/charting/src/components/StackedBarChart/StackedBarChart.styles.ts

```diff
@@ -60,6 +60,20 @@ export const getStyles = (props: IStackedBarChartStyleProps): IStackedBarChartSt
       fontSize: FontSizes.small,
       color: theme.palette.black,
       opacity: '0.6'
+    },
+    benchmarkContainer: {
+      position: 'relative',
+      height: '11.8px'
+    },
+    triangle: {
+      position: 'absolute',
+      width: '0',
+      height: '0',
+      borderLeft: '4.5px solid transparent',
+      borderRight: '4.5px solid transparent',
+      borderTop: '7.8px solid',
+      borderTopColor: theme.palette.neutralTertiary,
```

Comment:
Shouldn't this change with the passed in color?

Reply:
passed in benchmarkColor and targetColor to styles.